From b2ee66fe1224fe31380a87e290be18e0d480ccaf Mon Sep 17 00:00:00 2001
From: Stanislaw Gruszka <sgruszka@redhat.com>
Date: Wed, 10 Dec 2014 10:29:26 -0500
Subject: [wireless] iwlwifi/mvm: disable BT Co-running by default

Message-id: <1418207389-7899-13-git-send-email-sgruszka@redhat.com>
Patchwork-id: 101586
O-Subject: [RHEL7.1 PATCH 12/35] iwlwifi: mvm: disable BT Co-running by default
Bugzilla: 1155538
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: John Greene <jogreene@redhat.com>

From: Emmanuel Grumbach <emmanuel.grumbach@intel.com>

BZ#1155538

commit 9b60bb6d86496af1adc753795de2c12c4499868a upstream.

The tables still contain dummy values.

Signed-off-by: Emmanuel Grumbach <emmanuel.grumbach@intel.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Brew:
https://brewweb.devel.redhat.com/taskinfo?taskID=8330268

Testing:
Tested on iwl7265 adapter.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/wireless/iwlwifi/mvm/constants.h b/drivers/net/wireless/iwlwifi/mvm/constants.h
index 5168569..cb4c06c 100644
--- a/drivers/net/wireless/iwlwifi/mvm/constants.h
+++ b/drivers/net/wireless/iwlwifi/mvm/constants.h
@@ -80,7 +80,7 @@
 #define IWL_MVM_WOWLAN_PS_SNOOZE_WINDOW  25
 #define IWL_MVM_LOWLAT_QUOTA_MIN_PERCENT 64
 #define IWL_MVM_BT_COEX_SYNC2SCO  1
-#define IWL_MVM_BT_COEX_CORUNNING  1
+#define IWL_MVM_BT_COEX_CORUNNING  0
 #define IWL_MVM_BT_COEX_MPLUT   1
 
 #endif /* __MVM_CONSTANTS_H */
-- 
1.7.1